wsd/Storage.hpp

```diff
@@ -532,8 +533,14 @@ class LocalStorage : public StorageBase
     }
 
     void updateLockStateAsync(const Authorization&, LockContext&, LockState, const Attributes&,
-                              const AsyncLockStateCallback&) override
+                              SocketPoll&,
+                              const AsyncLockStateCallback& asyncLockStateCallback) override
     {
+        if (asyncLockStateCallback)
+        {
+            asyncLockStateCallback(AsyncLockUpdate(AsyncLockUpdate::State::Complete,
+                                                   LockUpdateResult(LockUpdateResult::Status::OK)));
+        }
```

Comment:
Is this a temporary patch to be completed at a later time?
- Why is this just claiming AsyncLockUpdate::State::Complete, disregarding the LockState?
- Why is LockState (and others) ignored? LockState and AsyncLockUpdate::State have different semantics.

Reply:
Capturing the lock state is added later, in a separate commit. If you looked at individual commits, it might not have been there. Notice that both your comments saying 'it's missing' and, later, that 'it's provided' are on the same function.

I also checked and couldn't find a case that was missing the lock state.